### ISSUE TITLE
Stacked line charts

### ISSUE THREAD

**Issue by timgl** [PostHog/posthog#1167]
**Is your feature request related to a problem? Please describe.**

When I'm breaking down a trends graph by a property, that one line splits into many lines. A logical way to represent this would be a stacked line chart

**Describe the solution you'd like**

Add a stacked line chart option https://codepen.io/natenorberg/pen/WwqRar

**Describe alternatives you've considered**

Use the overlapping lines

**Additional context**

***Thank*** you for your feature request - we love each and every one!

**Comment by mariusandra** [comment]
Stacked bar charts would be nice too: https://www.chartjs.org/docs/latest/charts/bar.html#stacked-bar-chart

**Comment by timgl** [comment]
Yep, and normal bars too. Those should all be relatively easy to do in one go

**Comment by Kacppian** [comment]
I'd like to pick this up.

**Comment by timgl** [comment]
Woop! Go for it :)

**Comment by pauldambra** [comment]
@thmsobrmlr implemented area line charts
